models/demos/t3000/falcon40b/tt/falcon_decoder.py

```diff
@@ -205,7 +205,7 @@ def fwd_prefill(
             )
 
         if replicated_hidden_states.dtype != self.model_config["BFP8_DTYPE"]:
-            replicated_hidden_states = ttnn.experimental.typecast(
+            replicated_hidden_states = ttnn.typecast(
```

Comment:
this can to stay ttnn.experimental.typecast. it refers to typecast in ttnn/operations/experimental/copy